src/Processors/Formats/Impl/VerticalRowOutputFormat.cpp

```diff
@@ -141,7 +141,7 @@ void VerticalRowOutputFormat::writeSpecialRow(const Columns & columns, size_t ro
     row_number = 0;
     field_number = 0;
 
-    size_t num_columns = columns.size();
+    size_t columns_size = columns.size();
```

Comment:
Why do we need such renaming?

Reply:
Because num_columns is class member.

Reply:
It's interesting how compilers choose to differ in such a common warning.